pkg/apis/ocs/v1/storagecluster_types.go

```diff
@@ -43,7 +43,7 @@ type StorageClusterSpec struct {
 	// If enabled, sets the failureDomain to host, allowing devices to be
 	// distributed evenly across all nodes, regardless of distribution in zones
 	// or racks.
-	FlexibleScaling  bool     `json:"flexibleScaling,omitempty"`
+	FlexibleScaling bool `json:"flexibleScaling,omitempty"`
```

Comment:
Move out of this commit.

Reply:
This is an automatic change done by running `gofmt` (which I run before every commit)

Reply:
That's fine, please move it into the tidy commit.

Reply:
done